library/ssl_tls.c

```diff
@@ -574,11 +573,9 @@ static int tls_prf_sha384( const unsigned char *secret, size_t slen,
                              label, random, rlen, dstbuf, dlen ) );
 }
 #endif /* MBEDTLS_SHA384_C */
-#endif /* MBEDTLS_SSL_PROTO_TLS1_2 */
 
 static void ssl_update_checksum_start( mbedtls_ssl_context *, const unsigned char *, size_t );
 
-#if defined(MBEDTLS_SSL_PROTO_TLS1_2)
```

Comment:
Here in TLS 1.3 I'd say we use only `ssl_update_checksum_sha256`, thus better to keep the other ones under the MBEDTLS_SSL_PROTO_TLS1_2 guard.